apps/camera/js/lib/camera/camera.js

```diff
@@ -951,6 +966,43 @@ Camera.prototype.startedRecording = function() {
   this.set('recording', 'started');
 };
 
+Camera.prototype.pauseRecording = function() {
+  debug('pause recording');
+
+  var state = this.get('recording');
+  if (state !== 'started' && state !== 'resumed') {
+    debug('not recording, cannot pause');
+    return;
+  }
+
+  this.mozCamera.pauseRecording();
+  this.set('recording', 'pausing');
+};
+
+Camera.prototype.pausedRecording = function() {
```

Comment:
I know there's no convention currently for this, but if would be nice if things like this that are essentially callbacks were named clearer as such. Like `onRecordingPaused`.